### PR TITLE
Update manual testing section in DG

### PR DESCRIPTION
**Pull request by FeliciaBeatrice**

### CODE REVIEW

**Review by OVReader — COMMENTED** [review]
Informative! LGTM

**Review by zoranabc201 — APPROVED** [review]
Looks comprehensive. Good job!

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -926,6 +964,33 @@ java -jar PropertyRentalManager.jar
...
+2. Unsuccessful check client
+    1. Test case: `check -client i/0`
+
+       Expected: Terminal shows error message.
+
+    2. Test case: `check -client i/[INDEX]`, where INDEX is an index that is not in the client list (1-indexed).
+
+       Expected: Terminal shows error message.
+
+    3. Test case: `check -client i/hello`
```
Comment (wilsonngja) [inline docs/DeveloperGuide.md:990]: Good job including the expected output.

**Review by wilsonngja — APPROVED** [review]
LGTM. Thanks for updating

**Pull request merged**